Makefile

```
@@ -53,13 +53,15 @@ pull_translations:
 	rm -rf src/i18n/messages
 	mkdir src/i18n/messages
 	cd src/i18n/messages \
-      && atlas pull --filter=$(transifex_langs) \
+      && atlas pull $(ATLAS_OPTIONS) \
+               translations/frontend-platform/src/i18n/messages:frontend-platform \
                translations/paragon/src/i18n/messages:paragon \
                translations/frontend-component-header/src/i18n/messages:frontend-component-header \
                translations/frontend-component-footer/src/i18n/messages:frontend-component-footer \
+			   translations/frontend-lib-special-exams/src/i18n/messages:frontend-lib-special-exams \
```

nit: formatting (mixed tabs/spaces)

```suggestion
               translations/frontend-lib-special-exams/src/i18n/messages:frontend-lib-special-exams \
```

---

Good catch. My editor fails me occasionally on mixed spacing types.

Now fixed.